data/json/snippets/lab.json

```diff
@@ -26,6 +26,242 @@
"\n\n\"Fuck off.\"\n\n\"You are in posession of a fake drivers's license issued to one Jonathan Smith, which is the alias you applied for employment with in our facility.  We will find out your real name in time, but I'll refer to you as Mr. Smith for this interview.  Is that acceptable, Mr. Smith?\"\n\n\"Fuck off.  I want to talk to my lawyer.\"\n\n\"You don't have a lawyer.\"\n\n\"I know my rights, shithead.  You won't get a word out of me before I talk to my lawyer.\"\n\n\"You seem to be confused about your situation, Mr. Smith.  As far as the world is concerned you stopped existing the moment you entered this facility, and as far as we are concerned you stopped having rights as soon as you tried to access areas above your provisional clearence.  Let me present your options going forward:  You continue witholding information from us and you will suffer a meticulously-documented accident leaving no doubt to your demise even though no body will ever be found.  Or you cooperate fully and without hesitation, in which case you will get to see the sky again and act like nothing has happened.  Am I understood?\"\n\n\"…yes.\"\n\n\"Please state your name for the record.\"\n\n\"M-my name is Thomas Wright.  I'm a, uh, a freelance journalist.\"\n\n\"What were you trying to accomplish in this facility, Mr. Wright?\"\n\n\"There was th-this guy offering good money for some pictures from the inside of this place.\"\n\n\"I don't suppose this man told you his name?\"\n\n\"No, no…He just handed me the application forms and the licence, paid quarter up front, and said to bring the pictures to the same spot in a week's time.\"\n\n\"What did he say to expect here?\"\n\n\"This place is…Look, you can't expect people not to talk about it, right?  You can't just put up a warehouse of a shipping company nobody's heard of practically overnight and act like it was always here.  He said you guys were a, uh, top secret prison for political prisoners.\"\n\n\"That explains your attempts to clumsily pick the first closed door you came across.  How did your employer contact you?\"\n\n\"A friend of a friend gave him my number, at least according to him.  If you haven't noticed in here times are tough, so I couldn't really afford to be picky about work even before he started throwing cash around.\"\n\n\"Very well.  While we verify that accout my colleagues here will take you through a tour of our facility:  You will take photographs with your \"hidden\" camera when and where they instruct you to.  You will submit these pictures for review and remain in our custody until we can be certain of your compliance.  You will be released and at the appointed time you will hand over the pictures to your employer.  You will not mention this conversation to them, or to anybody outside of this room.  You will enjoy your well-earned payment, far away from this facility.  Am I understood?\"\n\n\"Yes, sir, I can do that.\"\n\n\"Excellent, Mr. Wright.  I look forward to never seeing you again.\""
```

Comment:
Translators reported that there are multiple typos in this "lab interview log". Are those intentional for atmosphere?

Reply:
Not really, this should be a direct transcript. Thanks for the heads-up, I'll give it a reread.